src/Host/Client/Impl/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
     "MicroBuild.Core": "0.2.0",
+    "Microsoft.AspNet.WebApi.Core": "5.2.3",
```

Comment:
This is a non-ASP.NET Core assembly (it's from an older ASP.NET version). What is it needed for?